content/millennium/r4/conformance/structure-definition.md

```diff
@@ -106,6 +106,7 @@ Authorization is not required.
  [`performing-location`]                                     | A location where the resource was performed.
  [`period`]                                                  | A time period defined by a start and end date/time.
  [`pharmacy-verification-status`]                            | Represents whether a MedicationRequest has been verified by a pharmacist.
+ [`priority`]                                                | The priority of the field or element that is a list.
```

Comment:
I thought the definition that we landed was "The priority of the element within a list."

Reply:
Updated - https://github.com/cerner/fhir.cerner.com/pull/639/commits/f39ebcb7f8d1d494cd811d7f0eaee9367b87b8e2#diff-f10dc1eb3735aa1264e82ae17b8a0cd967fc69912568786cd6406be319d3fd17R109